src/server/api/routers/commission.ts

```
@@ -62,168 +59,277 @@ export const commissionRouter = createTRPCRouter({
+                .refine((data) => (data.slug !== undefined) !== (data.id !== undefined), {
+                    message: 'Either slug or id must be provided, but not both'
+                })
         )
         .query(async ({ input, ctx }) => {
             const clerk = await clerkClient()
-            const data = await ctx.db.query.artists.findFirst({
-                where: eq(artists.handle, input.handle),
+            const commission = await ctx.db.query.commissions.findFirst({
+                where: input.slug
+                    ? eq(commissions.slug, input.slug)
+                    : eq(commissions.id, input.id!),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)